cmd/cat/cat.go

```diff
@@ -11,126 +10,65 @@ import (
 	"github.com/nlnwa/warchaeology/internal/filter"
 	"github.com/nlnwa/warchaeology/internal/flag"
 	"github.com/spf13/viper"
-
-	"github.com/spf13/cobra"
 )
 
-type conf struct {
-	offset             int64
-	recordNum          int
-	recordCount        int
-	fileName           string
-	filter             *filter.Filter
-	showWarcHeader     bool
-	showProtocolHeader bool
-	showPayload        bool
+type catConfig struct {
+	Offset             int64
```

Comment:
Since the struct is only used in the `cat` package the fields need not be public.

Reply:
Makes sense, will fix.